homeassistant/components/hue/bridge.py

```diff
@@ -77,9 +78,19 @@ def host(self):
             DOMAIN, SERVICE_HUE_SCENE, self.hue_activate_scene, schema=SCENE_SCHEMA
         )
 
+        self.parallel_updates_semaphore = asyncio.Semaphore(
+            3 if self.api.config.modelid == "BSB001" else 10
+        )
+
         self.authorized = True
         return True
 
+    async def async_request_call(self, coro):
+        """Process request batched."""
+
+        async with self.parallel_updates_semaphore:
+            await coro
```

Comment:
let's return the result, in case someone needs it.
```suggestion
            return await coro
```